apis/refs/v1beta1/secretmanagerrefs.go

```diff
@@ -145,6 +145,9 @@ func ResolveSecretManagerSecretVersionRef(ctx context.Context, reader client.Rea
 
 	// External should be in the "projects/*/secrets/*/versions/*" format
 	if ref.External != "" {
+		if ref.External == "latest" {
+			return ref, nil
+		}
```

Comment:
Found this special case from the the `RunJob` CRD `versionRef` description,

Reply:
What does it mean?  Which project?  Which secret?

Reply:
This refers to the SecretManagerSecretVersion, whose parent (SecretMangerSecret) is also required in RunJob. So it would use the same as the parent. Also, this is a [GCP server ask] field(https://cloud.google.com/run/docs/configuring/jobs/secrets#terraform_1), not a TF special handling (I verified in the KCC TF-based reconciler code base). I think the real problem is that this field should be `version`, but the current API requires `versionRef` and its parentRef.